src/main/scala/mesosphere/marathon/core/matcher/util/ActorTaskLaunchSource.scala

```diff
@@ -0,0 +1,25 @@
+package mesosphere.marathon.core.matcher.util
```

Comment:
Hmmm. Good question. We could discuss this.

Basically, this code helps implementors of the top-level traits. Implementors do not have to use the code though. So it helps but is not required.

I would like to restrict the top-level packages to the interfaces but maybe there is a better name than util? Does that make sense to you or not?